sdcm/nemesis.py

```diff
@@ -1392,7 +1392,6 @@ def disrupt_terminate_kubernetes_host_then_replace_scylla_node(self):
     def _disrupt_kubernetes_then_replace_scylla_node(self, disruption_method):
         if not self._is_it_on_kubernetes():
             raise UnsupportedNemesis('Supported only on kubernetes')
-        self.set_target_node()
```

Comment:
Why removing it here ?

Reply:
I.e it's not clear where is the duplicate

Reply:
The main nemesis logic sets it before giving control to a nemesis.